src/glview/VertexStateManager.cc

```diff
@@ -0,0 +1,49 @@
+#include "VertexStateManager.h"
+
+void VertexStateManager::initializeSize(size_t vertices_size) {
+  std::vector<GLuint> placeholder = std::vector<GLuint>();
```

Comment:
Explicit initialization unnecessary as it does the same as default initialization.